src/display/editor/editor.js

```diff
@@ -840,6 +850,52 @@ class AnnotationEditor {
     }
   }
 
+  #setAltTextButtonState() {
+    const button = this.#altTextButton;
+    if (!button) {
+      return;
+    }
+    if (this.#altTextDecorative) {
+      button.classList.add("done");
+      button.title = "";
+      if (this.#altTextAriaDescription) {
+        button.removeAttribute("aria-describedby");
+        this.#altTextAriaDescription.remove();
+        this.#altTextAriaDescription = null;
+      }
+    } else if (this.#altText) {
+      button.classList.add("done");
+      button.title = this.#altText;
+      let description = this.#altTextAriaDescription;
+      if (!description) {
+        this.#altTextAriaDescription = description =
+          document.createElement("span");
+        description.className = "description";
+        const id = (description.id = `${this.id}-alt-text`);
+        button.append(description);
+        button.setAttribute("aria-describedby", id);
```

Comment:
It's too bad that `aria-description` doesn't appear to be supported yet, since it seems that it'd simplify things here (by not requiring a separate element); see https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-description

Maybe add a TODO-comment about using that once it becomes available?